examples/analytics-vendors.amp.html

```diff
@@ -27,6 +27,44 @@
 <div id="container">
 Container for analytics tags. Positioned far away from top to make sure that doesn't matter.
 
+<!-- Acquia Lift example -->
+<amp-analytics type="acquialift" id="acquialift">
++<!-- acquialift type -->
```

Comment:
There see to be two + here. Is that correct? Also, the comment seems redundant (or I do the know what it is for). Maybe remove it?